presto-product-tests/README.md

```diff
@@ -407,3 +407,14 @@ been downloaded:
     # downloaded.
     presto-product-tests/conf/docker/<profile>/compose.sh pull
     ```
+
+## Possible issues
```

Comment:
This is not limited to debugging only. You may not even be able to start tests if port 1080 is taken.